lake_import.py

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Bot to import and source statements about Surface Water Bodies.
+
+Author: Lokal_Profil
+License: MIT
+
+usage:
+    python lake_import.py [OPTIONS]
+
+&params;
+"""
+from __future__ import unicode_literals
+from builtins import dict
+
+import pywikibot
+
+import wikidataStuff.helpers as helpers
+from wikidataStuff.WikidataStuff import WikidataStuff as WdS
+
+from WFDBase import WfdBot, UnmappedValueError
+
+parameter_help = """\
+Lakebot options (may be omitted):
+-year              Year to which the WFD data applies.
+-new               if present new items are created on Wikidata, otherwise
+                   only updates are processed.
+-in_file           path to the data file
+-mappings          path to the mappings file (if not "mappings.json")
+-cutoff            number items to process before stopping (if not then all)
+
+Can also handle any pywikibot options. Most importantly:
+-simulate          don't write to database
+-dir               directory in which user_config is located
+-help              output all available options
+"""
+docuReplacements = {'&params;': parameter_help}
+EDIT_SUMMARY = 'importing #SWB using data from #WFD'
+
+
+class LakeBot(WfdBot):
+    """Bot to enrich/create info on Wikidata for lake objects."""
```

Comment:
I think it's better to use the term Surface Water Bodies instead of lakes throughout, to avoid confusion.